geoips/plugins/modules/readers/amsr2_netcdf.py

```diff
@@ -390,3 +392,17 @@ def call(
         )
     final_xarrays["METADATA"] = list(final_xarrays.values())[0][[]]
     return final_xarrays
+
+
+def get_test_files():
+    """Generate test files for unit testing reader."""
+    filepath = os.environ["GEOIPS_TESTDATA_DIR"] + "/test_data_amsr2/data/AMSR2*.nc"
+    filelist = glob(filepath)[:2]
+    tmp_xr = call(filelist)
+
+    return tmp_xr
+
+
+def get_test_parameters():
+    """Generate a data key for unit testing."""
+    return {"data_key": "Brightness_Temperature_10_GHzH", "data_var": "tb10h"}
```

Comment:
Standardized naming

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/375#discussion_r1372485870
Resolved that conversation